build.assets/Makefile

```diff
@@ -201,7 +201,7 @@ test: buildbox
 		$(DOCKERFLAGS) $(BCCFLAGS) $(NOROOT) -t $(BUILDBOX) \
 		/bin/bash -c \
 		"examples/etcd/start-etcd.sh & sleep 1; \
-		gcloud beta emulators firestore start --host-port=localhost:8618 & sleep 1; \
```

Comment:
Per chesterson's fence, is it safe to remove the sleep?  This seems unrelated to the other changes.

Reply:
You make a good point. I thought I'd remove it as it wasn't doing anything, but doing so meant that the emulator doesn't start in the background and so the tests don't actually run! Restored in da52cbd.